students/Tianx/Lesson3/src/basic_operations.py

```diff
@@ -1,27 +1,112 @@
+import logging
 from peewee import *
+from customer_model import Customer
 
-# add_customer(customer_id, name, lastname, home_address, phone_number, email_address, status, credit_limit): This function will add a new customer to the sqlite3 database.
-#
-# search_customer(customer_id): This function will return a dictionary object with name, lastname, email address and phone number of a customer or an empty dictionary object if no customer was found.
-#
-# delete_customer(customer_id): This function will delete a customer from the sqlite3 database.
-#
-# update_customer_credit(customer_id, credit_limit): This function will search an existing customer by customer_id and update their credit limit or raise a ValueError exception if the customer does not exist.
-#
-# list_active_customers(): This function will return an integer with the number of customers whose status is currently active.
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
 
+db = SqliteDatabase('customers.db')
+db.create_tables([Customer])
+
+# pylint: disable = W0614, W0401, C0301, C0114, W1203
```

Comment:
The disable only applies to code lines after the location of the disable, try moving it to line 1 and run Pylint again.